test/powershell/Modules/Microsoft.PowerShell.Utility/Get-FormatData.Tests.ps1

```diff
@@ -15,7 +15,9 @@ Describe "Get-FormatData" -Tags "CI" {
         $format.TypeNames | Should -HaveCount 2
```

Comment:
@adityapatwardhan, your last commit had 54 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:288 char:22
+             ${psObj}?.name | Should -BeExactly 'value'
+                      ~~~~~
Unexpected token '.name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:289 char:22
+             ${array}?.length | Should -Be 3
+                      ~~~~~~~
Unexpected token '.length' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:290 char:21
+             ${hash}?.a | Should -Be 1
+                     ~~
Unexpected token '.a' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:292 char:36
+             (Get-Process -Id $pid)?.Name | Should -BeLike "pwsh*"
+                                    ~~~~~
Unexpected token '.Name' in expression or statement.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~
You must provide a value expression following the '?' operator.

At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Language/Operators/NullConditional.Tests.ps1:293 char:35
+             (Get-Item $TestDrive)?.EnumerateFiles()?.Name | Should -B ?
+                                   ~~~~~~~~~~~~~~~
Unexpected token '.EnumerateFiles' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Built-in type information tests.Should have correct number of built-in type items in type table
```powershell
Expected exactly 273, but got 271.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/ETS/TypeTable.Tests.ps1: line 22
22:         $types.Count | Should -BeExactly ($isUnixStatEnabled ? 272 : 273)
```
Tests for $ErrorView.NormalView tests.Error shows up when using strict mode
```powershell
Expected like wildcard '*Oops!*' to match '
', but it did not match.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/Formatting/ErrorView.Tests.ps1: line 82
82:             $e | Should -BeLike '*Oops!*'
```